core/src/main/java/me/lihq/game/ScenarioBuilder.java

```diff
@@ -119,6 +120,9 @@ Set the first clues in the list to red herrings (the number of red herrings
 
             if (randHidingSpot != null) {
                 room.addClue(clue.setTileCoordinates(randHidingSpot));
+            } else
+            {
+                System.out.println("Failed to place Clue " + clue.getName() + " in room " + room.getName());
```

Comment:
cool, no just wanted to make sure it wasnt for popping up in the game or anything